Integrations/integration-Verodin.yml

```diff
@@ -0,0 +1,859 @@
+                res.push(createContextSingle(data[j]));
+            }
+            return res;
+        }
+        return createContextSingle(data);
+    }
+
+    function mapObjFunction(mapFields, filter) {
+        var transformSingleObj= function(obj) {
+            var res = {};
+            mapFields.forEach(function(f) {
+                res[f.to] = dq(obj, f.from);
+            });
+            if (filter && !filter(res)) {
+                return undefined;
+            }
+            return res;
+        };
+        return function(obj) {
+            if (obj instanceof Array) {
+                var res = [];
+                for (var j=0; j < obj.length; j++) {
+                    var current = transformSingleObj(obj[j]);
+                    if (current) {
+                        res.push(current);
+                    }
+                }
+                return res;
+            }
+            return transformSingleObj(obj);
+        };
+    }
+
+    var sendRequest = function(url, method, args) {
+        var headers = {};
+        var httpParams = {
+           Method: method,
+           Username: params.credentials.identifier,
+           Password: params.credentials.password,
+        };
+        var res = http(
+                baseUrl + url + (method === 'GET' && method ? encodeToURLQuery(args) : ''),
+                httpParams,
+                params.insecure,
+                params.proxy,
+                false
+            );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request ' + url + ' failed, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+        }
+        if (res.StatusCode === 204) {
+            return undefined;
+        }
+        try {
+            return JSON.parse(res.Body);
+        }
+        catch (ex) {
+            throw 'Failed to create JSON from response ' + res.Body + ', exception is ' + ex;
+        }
+    }
+
+    var toArray = function(obj) {
+        res = [];
+        keys = Object.keys(obj);
+        for (var k in keys) {
+            res.push(obj[keys[k]]);
+        }
+        return res;
+    }
+
+    var createIncidentFromJob = function(incident) {
+        var keys = Object.keys(incident);
+        var labels = [];
+        for (var i = 0; i<keys.length; i++) {
+            labels.push({'type': keys[i], 'value': String(incident[keys[i]])});
+        }
+        return {
+            "name": incident.id + ' ' + incident.desc,
+            "labels": labels,
+            "rawJSON": JSON.stringify(incident),
+        }
+    }
+
+    if (command === 'fetch-incidents') {
+        var lastRun = getLastRun();
+        var lastID = lastRun && lastRun.startID ? lastRun.startID : 0;
+        var startID = lastID;
+        currentCommand = commandDictionary['verodin-get-jobs'];
+        var res = sendRequest(replaceInTemplatesAndRemove(currentCommand.url, args), currentCommand.method, args);
```

Comment:
Where are you limiting the fetched jobs according to startID? 
Getting all jobs and filtering after the fact might cause problems on large scales